extended/src/main/java/io/kubernetes/client/extended/kubectl/util/selector/SelectorHelper.java

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.extended.kubectl.util.selector;
+
+import io.kubernetes.client.openapi.models.V1LabelSelector;
+import io.kubernetes.client.openapi.models.V1LabelSelectorRequirement;
+import io.kubernetes.client.util.labels.EqualityMatcher;
+import io.kubernetes.client.util.labels.LabelMatcher;
+import io.kubernetes.client.util.labels.LabelSelector;
+import io.kubernetes.client.util.labels.SetMatcher;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class SelectorHelper {
```

Comment:
wdyt if we move this helper into the existing static helper class `LabelSelector`?